k8s/anti-fraud.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: OTEL_EXPORTER_OTLP_ENDPOINT
           value: "http://lgtm:4318"
         - name: OTEL_RESOURCE_ATTRIBUTES
-          value: service.name=anti-fraud,service.namespace=shop,service.version=1.1,deployment.environment=staging
+          value: service.name=anti-fraud,service.namespace=shop,service.version=1.1,deployment.environment.name=staging
```

Comment:
@zeitlinger can we migrate the Helm chart to `deployment.environment.name`?

Reply:
yes, good idea